From e85862774ef049227653310e8bfc3838e015ef0d Mon Sep 17 00:00:00 2001
From: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
Date: Fri, 30 Oct 2015 18:11:21 +0200
Subject: [PATCH 292/452] fsl-dpaa2: mac: Remove unused function prototype

fixed_phy_register_2() was a leftover since we had to backport fixed PHY
implementation on kernel v3.10.

Signed-off-by: Bogdan Hamciuc <bogdan.hamciuc@freescale.com>
[Xulin: Original patch taken from FSL LS2085 SDK EAR6.0,
LS2085A-SDK-SOURCE-20160304-yocto.iso]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/mac/mac.c |    5 -----
 1 files changed, 0 insertions(+), 5 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index e30f4ab..826c499 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -63,11 +63,6 @@
 #define ppx_info(netdev, ...) dev_err(&netdev->dev, __VA_ARGS__)
 #endif /* CONFIG_FSL_DPAA2_MAC_NETDEVS */
 
-struct phy_device *fixed_phy_register2(unsigned int irq,
-				       struct fixed_phy_status *status,
-				       struct device_node *np);
-
-
 struct ppx_priv {
 	struct net_device		*netdev;
 	struct fsl_mc_device		*mc_dev;
-- 
1.7.5.4

